base/src/main/java/com/austindroids/austinfeedsme/data/FilterableEventsRepository.kt

```diff
@@ -0,0 +1,23 @@
+package com.austindroids.austinfeedsme.data
+
+import io.reactivex.Observable
+import javax.inject.Singleton
+
+@Singleton
+open class FilterableEventsRepository(private val eventsRemoteDataSource: FilterableEventDataSource)
+    : EventsRepository(eventsRemoteDataSource), FilterableRxEventsDataSource {
+
+    override fun getEventsRX(onlyFuture: Boolean, onlyFood: Boolean): Observable<List<Event>> {
+        return Observable.create {
```

Comment:
If this was my project, I wouldn't let this in until `it` is explicitly named `emitter`.